readme update (#25)
kanopy-platform/cdnvalidator: README.md

# cdnvalidator service

[![Build Status](https://drone.corp.mongodb.com/api/badges/kanopy-platform/cdnvalidator/status.svg)](https://drone.corp.mongodb.com/kanopy-platform/cdnvalidator)

> The cdnvaildator provides multi-tenant entitlement abstraction layer into CloudFront invalidations which may have different owners, performance, and cache control requirements.

Reviewer: nit: spelling, and I think this flows a bit better to breaks up the terse sentence:

```suggestion
The cdnvalidator provides an abstraction layer for CloudFront invalidations. It provides control for performance, different owners of distribution paths, and cache control requirements.
```

Author: I took most of this, but changed some of it.

## Architecture

> The CDN Vaildator provides a RESTful API detailed in the [OpenAPI documentation](./swagger/swagger.json).

Reviewer: spelling: `Validator`

### Authn/Authz

> The service relies on being deployed behind a reverse proxy that handles forward authentication and validation of the JWT.  Authorization decisions will be made against the `groups` and `scps` claims of a JWT.

Reviewer: nit: is the literal `scp` instead?
